mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -322,6 +329,8 @@ CLASS none/sg net/minecraft/entity/Entity
 		ARG 0 entity
 	METHOD r isTeammate (Lnone/sg;)Z
 		ARG 0 entity
+	METHOD s isEqualEntity (Lnone/sg;)Z
```

Comment:
The param type here is `Entity`, so I don't want to override `isEqual`.
It returns true if the param == this, or if this is an entity part and the param is the entity that owns that part.